docs/development/process.md

```diff
@@ -36,3 +39,5 @@ The validation process for new releases usually takes a couple of days and inclu
 1. The "release responsible" is verifying new features or other notable changes (derived of the draft release notes) in this development system.
 
 If all tests are green, all checks were successful, and the release responsible has performed all of the planned verifications then the release is triggered.
+
+> [*] This can be seen as "code freeze" period for the `vX.Y.0` release. The development and pull request merges to `master` can continue. Please note that you would have to pro-actively creates pull requests to the respective `release-vX.Y` branch in case you want to have your change that was already merged to `master` included in the `vX.Y.0` release.
```

Comment:
```suggestion
> [*] This can be seen as "code freeze" period for the `vX.Y.0` release. The development and pull request merges to `master` can continue. Please note that you would have to pro-actively create pull requests to the respective `release-vX.Y` branch in case you want to have your change that was already merged to `master` included in the `vX.Y.0` release. We mostly accept bug fixes only at this time, since a major part of the code has already been verified.
```